cmd/version.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"runtime/debug"
+	"sync"
+
+	"github.com/go-flutter-desktop/hover/internal/log"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print Hover version information",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) > 0 {
+			return errors.New("No arguments allowed")
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		version := hoverVersion()
+		fmt.Printf("Hover %s\n", version)
+	},
+}
+
+var (
+	hoverVersionValue string
+	hoverVersionOnce  sync.Once
+)
+
+func hoverVersion() string {
+	hoverVersionOnce.Do(func() {
+		buildInfo, ok := debug.ReadBuildInfo()
+		if !ok {
+			log.Errorf("Cannot obtain version information from hover build. To resolve this, please go-get hover using Go 1.13 or newer.")
+			os.Exit(1)
```

Comment:
Most of the time, when people run `go get ... hover` they install the `@master` version of hover, and not the `@latest`. This is due because not everyone have done the switch to go module. 

We should to the error string the command that they need to run to properly install hover.
If I'm not mistaking it's: `GO111MODULE=auto go get -u github.com/go-flutter-desktop/hover`.

@jld3103 we should also start to make sure everything is properly tagged. Our last tag was kinda old.